src/Cameras/VR/babylon.vrExperienceHelper.ts

```diff
@@ -921,7 +921,7 @@ module BABYLON {
                 }
 
                 this.raySelectionPredicate = (mesh) => {
-                    return mesh.isVisible;
+                    return mesh.isVisible && mesh.isPickable;

```

Comment:
@davrous is there a reason this was not in this predicate by default? Would this be okay to add now?

Reply:
Agree this should be like that (with the isPickable)

Reply:
Indeed, this is better like that. Thanks!